web/src/components/MapView.tsx

```diff
@@ -94,6 +94,7 @@ const MapView = (props: MapViewProps): ReactElement => {
   return (
     <MapContainer>
       <ReactMapGL
+        reuseMaps
```

Comment:
https://visgl.github.io/react-map-gl/docs/get-started/tips-and-tricks#minimize-cost-from-frequent-re-mounting